packages/web/examples/ssr/pages/index.js

```diff
@@ -101,7 +102,15 @@ const components = {
 	},
 };
 
+const Wrapper = props => {
+	return <DataSearch {...props} />;
+};
+
 export default class Main extends Component {
+	state = { mount: false };
+	componentDidMount() {
+		this.setState({ mount: true });
+	}
```

Comment:
Shouldn't this be a part of library? As on SSR this will break for sure because `window` will be undefined. Any specific reason we are doing this on example ? 

cc @lakhansamani @bietkul What do you guys think ?

Reply:
Yes, this should be part of the library. We in fact do something similar in ReactiveMaps as we cannot have maps rendered on the server. We should have checked there itself like,

```
if (window) {
   ....
}

```

Reply:
OK